### DISCUSSION TITLE
Lazer import from Stable in limited space conditions?

### DISCUSSION THREAD

**Discussion by jsongalvez** [ppy/osu#22894] [Q&A] [answered]
I have a 256GB SD card and osu! takes 200GB + other files leaving me with 48GB. One way would be an option for lazer to delete the item from stable after import.

**Comment by Loreos7** [comment, marked as answer]
Please read https://osu.ppy.sh/wiki/en/Client/Release_stream/Lazer/File_storage

**Reply by jsongalvez** [reply]
I had exfat on sdcard... Thanks.
